### ISSUE TITLE
Typings for static members are wrong

### ISSUE THREAD

**Issue by BickelLukas** [froala/wysiwyg-editor#4644]
In the `FroalaEditor` class the static members (like `KEYCODE`, `ENTER_DIV` etc.) are declared as properties on the class instance and are therefore not accessible as static properties.

##### Expected behavior.
`FroalaEditor.KEYCODE.Enter` should not throw a typescript error

##### Actual behavior.
`FroalaEditor.KEYCODE.Enter` throws an error

##### Steps to reproduce the problem.
```js
import FroalaEditor from "forala-editor";

const keyCode = FroalaEditor.KEYCODE.Enter; // <- Error
```

##### Editor version.
4.1.0

**Comment by johnnyoshika** [comment]
I'm getting so many type errors after upgrading to 4.1.0 and I believe it's related to this issue. I'll list examples below where `FroalaEditor` is aliased as `FE`:

```
import FE from 'froala-editor';
```

Examples that cause type errors:

```
FE.DEFAULTS
FE.DefineIcon
FE.DefineIconTemplate
FE.END_MARKER
FE.KEYCODE
FE.MARKERS
FE.PLUGINS
FE.POPUP_TEMPLATES
FE.RegisterCommand
FE.RegisterQuickInsertButton
FE.RegisterShortcut
FE.START_MARKER
```

**Comment by ilyaskarim** [comment]
Fix for this issue is being rolled out in V4.1.1: https://froala.com/wysiwyg-editor/changelog/#4.1.1
